orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/cluster/FindImageFromClusterTask.groovy

```diff
@@ -227,10 +228,23 @@ class FindImageFromClusterTask extends AbstractCloudProviderAwareTask implements
       }
     }.flatten()
 
+    List<Artifact> artifacts = deploymentDetails.collect {
+      new Artifact(
+        name: it.get("imageName"),
+        type: it.get("cloudProvider"),
```

Comment:
No worries, I didn't see the PR had been out as long as it has, I assumed it was submitted around the time gard pinged us.